vhost-device-console/src/main.rs

```diff
@@ -79,18 +83,24 @@ impl TryFrom<ConsoleArgs> for VuConsoleConfig {
             return Err(Error::WrongBackendSocket);
         }
 
+        if args.max_queue_size == 0 {
```

Comment:
IIRC queue_size should also be power of 2, btw will a 0 value fails later eventually?

Reply:
Will add a check for the power of 2. There are two ways to handle `0`: replacing with the default value, or throwing an error directly. I open to both ways. Which one do you prefer?

Reply:
> will a 0 value fails later eventually?

Yes, an error will be thrown if not check 0:

```
[2025-02-13T02:58:15Z ERROR vhost_device_console] Could not create daemon: cannot create vhost user handler: failed to create vring: invalid queue maximum size
```

Reply:
Yep, I agree!